Package.swift

```diff
@@ -220,7 +203,7 @@ let package = Package(
         // TODO: - Add support for cflags cSetting so that we can set the -fno-autolink option
       ],
       linkerSettings: [
-        .linkedFramework("UIKit", .when(platforms: [.iOS, .tvOS])),
+        .linkedFramework("UIKit", .when(platforms: [.iOS, .macCatalyst, .tvOS])),
```

Comment:
Note: the [`macCatalyst`](https://developer.apple.com/documentation/packagedescription/platform/maccatalyst) was previously unavailable.